### ISSUE TITLE
/api/comments/:id

### ISSUE THREAD

**Issue by kyujin-cho** [thy2134/JaramWorkshop#6]

**Comment by kyujin-cho** [comment]
# GET #
- Request: None
- Response
```
{
    "title": "Dfsd",
    "contents": "sfdfs",
    "name": "dfsgserfsg",
    "date": "fdsgsgfefr"
}
```

**Comment by kyujin-cho** [comment]
# PUT #
- Request: 
```

{
    "title": "SGefa",
    "contents": "fgs",
    "password": "dsfdsfsv"
}
```

- Response
1. If request was successful
```
{
    "success": true
}
```
2. If request was unsuccessful
```
{
    "success": false,
    "error": "Reason of failure"
}
```

**Comment by kyujin-cho** [comment]
# DELETE #
- Request: 
```

{
    "password": "dsfdsfsv"
}
```

- Response
1. If request was successful
```
{
    "success": true
}
```
2. If request was unsuccessful
```
{
    "success": false,
    "error": "Reason of failure"
}
```

**Comment by kyujin-cho** [comment]
Moved GET, PUT, DELETE methods to /api/comments/:id route. 
this closes issue #6.
